python/qa/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+rm -rf ./output/*
+for ((i=1;i<=50;i++)); do
```

Comment:
It's not clear why you iterate over this 50x, but if the intent is to "warm up" the system for a benchmark, I will note that the process of closing the process, opening a new process, parsing the file, etc. will wipe all the caches and throw off your benchmarks.

If that is the intent, it's probably worth just running TSP-Quantitative-Algorithm.py once, and putting a loop around `performSimulatedAnnealing` in SA-Benchmark.py.

Reply:
Hi Lane, thank you for the feedback. I have created this script to simply the collection and the generation of the trial sample (with n=50 population size). The code writes the algorithm output variables (i.e. values for the best solution cost, total execution time and the initial solution distance) for each completed execution in a text file. Those values were them analyzed using a statistical test (details in the README.md file). I can rename to improve clarity but the reason here is to have an automated why to deploy the proposed method and the benchmark to obtain a data set that can be quantified by statistical modeling. I have choose 50 arbitrarily as the minimum recommendation for the t-test is a sample size larger than 30.